src/canvas/Point.tsx

```diff
@@ -19,6 +19,10 @@ export class Point implements IPoint {
         return this._x === o.getX() && this._y === o.getY();
     }
 
+    isWithinRadius = (o: Point, radius: number): boolean => {
+        return Math.pow((o.getX() - this._x), 2) + Math.pow((o.getY() - this._y), 2) < Math.pow(radius, 2);
```

Comment:
You could use the distancesquared function on the left side of the inequality to avoid code reduplication

Reply:
Ohh yeah, thanks!